free-3.md

```diff
@@ -10,41 +10,126 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Venn diagrams display data in a cool and organized way. Many programs or apps 
+already exist for drawing Venn diagrams, but the ones I found don't adjust the 
+circles according to the text within. For example, some programs make the users 
+specify the exact size of the circles and data. Thus, it takes many tries to 
+format venn diagrams exactly as envisioned. But what happens if the data 
+changes? Then people would have to repeat this struggle.
+
+I want to design a DSL that, given text for each circle (and intersection of 
+circles), creates an image rendering of the venn diagram with the text snugly 
+inside the circles. Basically, circles are auto-formatted according to the text.
+
```

Comment:
A simple but unsolved problem. It could also be cool to visualize data in Venn diagrams since the size of circles are auto generated you could easily see which part of the Venn diagram is more significant.